components/AdditionalDatasheetEditor.tsx

```diff
@@ -259,9 +262,16 @@ export function AdditionalDatasheetEditor() {
         headerName: 'Baseline',
         field: 'baselineYear',
         flex: 1,
-        renderCell: (params: GridRenderCellParams) => (
-          <Typography variant="body2">{params.value}</Typography>
-        ),
+        renderCell: (params: GridRenderCellParams) => {
+          const value = params.value;
+          const formattedValue =
+            typeof value === 'number'
+              ? new Intl.NumberFormat(undefined, {
+                  maximumFractionDigits: 0,
+                }).format(value)
+              : value;
+          return <Typography variant="body2">{formattedValue}</Typography>;
+        },
```

Comment:
We'll need to format this with more decimal places for some measures. Ideally the backend would return this precision but for now we hack it on the frontend. 

Here's how we do that in the other datasheet editor:
https://github.com/kausaltech/nzc-data-studio/blob/056506fe22c5e97f4fa9cb20a9ea732d94d711ee/components/DatasheetEditor.tsx#L525-L539